### ISSUE TITLE
Error creating a tile +40+029 which I had created many times before.

### ISSUE THREAD

**Issue by Argaeus** [oscarpilote/Ortho4XP#124]
The problem started with a corrupt osm data which I seemingly corrected by revising it using JOSM. When I tried to proceed with tile creation afterwards, I started getting new error messages and I am not so literate in this area to solve the problem myself thus following the instructions to fila a "bug report".
![ortho4xp](https://user-images.githubusercontent.com/30521060/80278031-d82d9280-86fb-11ea-8ac2-2f0c5bcba19b.jpg)

[Data+40+029.zip](https://github.com/oscarpilote/Ortho4XP/files/4532814/Data%2B40%2B029.zip)

Best regards.

**Comment by d41k4n** [comment]
Your issue is not a bug but a topological inconsistency encountered during mesh creation resulting from bad OSM data which Ortho4XP wasn't able to handle. This requires analysis and fixing in OSM and is out of scope here.

I suggest doing a search for similar issues in the official Ortho4XP forum:
https://www.google.com/search?q=site%3A+forums.x-plane.org+ortho4xp+"Triangle4XP+really+couldn't+make+it"

**Comment by Argaeus** [comment]
Thanks for the reply Daikan. I found an older osm and elevation files in my other setup and I will try again using those. Rgds,Heyri.  Windows 10 için Posta ile gönderildi Kimden: d41k4nGönderilme: Saturday, April 25, 2020 2:10 PMKime: oscarpilote/Ortho4XPBilgi: Argaeus; AuthorKonu: Re: [oscarpilote/Ortho4XP] Error creating a tile +40+029 which I had created many times before. (#124) Your issue is not a bug but a topological inconsistency encountered during mesh creation resulting from bad OSM data which Ortho4XP wasn't able to handle. This requires analysis and fixing in OSM and is out of scope here.I suggest doing a search for similar issues in the official Ortho4XP forum:https://www.google.com/search?q=site%3A+forums.x-plane.org+ortho4xp+"Triangle4XP+really+couldn't+make+it"—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
